testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinatorTests.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections;
+using System.Linq;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace MLAPI.MultiprocessRuntimeTests
+{
+    [TestFixture(1)]
+    [TestFixture(2)]
+    public class TestCoordinatorTests : BaseMultiprocessTests
+    {
+        private int m_NbWorkers;
+        protected override int NbWorkers => m_NbWorkers;
```

Comment:
similar to my previous comment above, I think this needs a better name and /// <summary> block explaining it.